api/v01/vrp.rb

```diff
@@ -217,27 +216,27 @@ class Vrp < APIBase
               solution = OptimizerWrapper::Result.get(id)
               status 202
               if solution && !solution.empty?
-                output_format = params[:format]&.to_sym || (solution['csv'] ? :csv : env['api.format'])
+                output_format = params[:format]&.to_sym || (solution[:csv] ? :csv : env['api.format'])
                 if output_format == :csv
-                  present(OptimizerWrapper.build_csv(solution['result']), type: CSV)
+                  present(OptimizerWrapper.build_csv(solution[:result]&.collect{ |r| r.deep_stringify_keys }), type: CSV) # TODO : edit build_csv function to consider symbolized keys
```

Comment:
It is not very, but I might need to do it twice because of https://github.com/Mapotempo/optimizer-api/pull/32 (this PR moves build_csv logic to another file and clean functions).

But if this increases our memory issues then it may be worth to do it twice to avoid any problem !

Reply:
Hmm you are right there is quite a change in #32 and what is done here needs to be effectively duplicated there too :( 

I don't know what's the best but my gut feeling tells me that since replacing all string hash keys with symbol in the code is not that time consuming, it might worth to do it the right away.